src/endpoint/s3/ops/s3_put_object.js

```diff
@@ -37,7 +37,7 @@ async function put_object(req, res) {
         md_conditions: http_utils.get_md_conditions(req),
         source_md_conditions: http_utils.get_md_conditions(req, 'x-amz-copy-source-'),
         xattr: s3_utils.get_request_xattr(req),
-        xattr_copy: (req.headers['x-amz-metadata-directive'] === 'COPY'),
+        xattr_copy: (req.headers['x-amz-metadata-directive'] !== 'REPLACE'),
```

Comment:
What is the difference? There are only two variables COPY and REPLACE. Am I'm missing something?

Reply:
The case where it's not being sent, i.e. default behavior